yoshi/pkg/cni/cni.go

```diff
@@ -0,0 +1,35 @@
+package cni
+
+import (
+	"fmt"
+
+	plumberv1 "github.com/platform9/luigi/yoshi/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type CniProvider interface {
```

Comment:
Would be more convention to capitalize the acronyms.
```suggestion
type CNIProvider interface {
```